native-image-tests/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
   implementation project(':mockwebserver-deprecated')
   implementation project(':okhttp-urlconnection')
   implementation project(':mockwebserver-junit4')
-  testRuntimeOnly project(':mockwebserver-junit5')
+  implementation project(':mockwebserver-junit5')
```

Comment:
This one’s interesting. We don’t refer to it at compile time right?

Reply:
There is no test runtime!